src/prebid.js

```diff
@@ -1090,4 +1090,19 @@ $$PREBID_GLOBAL$$.processQueue = function () {
   processQueue($$PREBID_GLOBAL$$.cmd);
 };
 
+/**
+ * @alias module:pbjs.triggerBilling
+ */
+$$PREBID_GLOBAL$$.triggerBilling = (bid) => {
```

Comment:
what do you think about extracting [this piece of code](https://github.com/prebid/Prebid.js/blob/1dfae09702db70356ca373e085d578e0fac3314a/src/prebid.js#L998-L1013) and reusing it here? 

what I am worried about is that this call comes from the publisher and here `bid` could be anything at all; *but* the adapters will expect it to be an actual certified bid response object.

I would also improve that code a bit to look up by `bidId` as well. I don't like that it apparently accepts finding multiple matching bids, and just decides that the first one is the "correct" one.

Reply:
hey @dgirardi, i agree, makes sense to extract that piece of code and reuse it in the triggerBilling func, especially to your point about how a pub could pass through anything.

for the second part of your comment, if/when the `bids` var gets defined, i added an additional check to make sure that the bid we grab from that array has a matching transactionId with the bid that was passed through the triggerBilling func.  should that be sufficient?